src/mjolnir/pbfgraphparser.cc

```diff
@@ -731,6 +773,7 @@ struct graph_callback : public OSMPBF::Callback {
           case Use::kAlley:
           case Use::kEmergencyAccess:
           case Use::kDriveThru:
+          case Use::kLivingStreet:
```

Comment:
You set the surface based on use (i.e., kLivingStreet) but you never call set_use in OSMWay for kLivingStreet; therefore, it will default to a road.  This may have messed up your testing.  Not sure.